persistence/block.go

```diff
@@ -118,3 +129,35 @@ func (p *PostgresContext) insertBlock(block *coreTypes.Block) error {
 	_, err := tx.Exec(ctx, types.InsertBlockQuery(blockHeader.Height, blockHeader.StateHash, blockHeader.ProposerAddress, blockHeader.QuorumCertificate))
 	return err
 }
+
+func (p *PostgresContext) getCurrentAndNextValSetHashes() (currentValSetHash, nextValSetHash string, err error) {
+	// Get the next validator set
+	nextValSet, err := p.GetValidatorSet(p.Height)
+	if err != nil {
+		return "", "", err
+	}
+	nextValSetBz, err := codec.GetCodec().Marshal(nextValSet)
+	if err != nil {
+		return "", "", err
+	}
+	nValSetHash := crypto.SHA3Hash(nextValSetBz)
+	nextValSetHash = hex.EncodeToString(nValSetHash)
+
+	if p.Height == 0 {
+		return "", nextValSetHash, nil
+	}
+
+	// Get the current validator set
+	valSet, err := p.GetValidatorSet(p.Height - 1)
+	if err != nil {
+		return "", "", err
+	}
+	valSetBz, err := codec.GetCodec().Marshal(valSet)
+	if err != nil {
+		return "", "", err
+	}
+	valSetHash := crypto.SHA3Hash(valSetBz)
+	currentValSetHash = hex.EncodeToString(valSetHash)
```

Comment:
Move this into a small helper and just pass in the height - reuse for cu and next hashes